build/org.eclipse.pde.build.tests/src/org/eclipse/pde/build/internal/tests/ScriptGenerationTests.java

```diff
@@ -827,6 +827,7 @@ public void testBug248767_212467() throws Exception {
 	}
 
 	@Test
+	@Ignore("this test fails on some linux versions for unkown reasons")
```

Comment:
Instead of disabling it for all platforms it would be less invasive to just disable it for Linux using Assumptions.

Reply:
As said anyone that wants to invest time in pde-build and thinks this test is important can do so.

Reply:
That's OK for Linux, but why should it be disabled on other platforms?

Reply:
> Instead of disabling it for all platforms it would be less invasive to just disable it for Linux using Assumptions.

See https://github.com/eclipse-pde/eclipse.pde/pull/1152.